src/app/components/CallToActionLinkComponent/README.md

```diff
@@ -0,0 +1,111 @@
+| url                | string                         | The url for the Call to Action Link                                                                                                                                                                                                                                          |
+| children?          | node                           | The content included between the opening and closing tags of the Call to Action Link                                                                                                                                                                                         |
+| alignWithMargin?   | boolean                        | Applies display styling to align link with text. (Left margin for LTR, right margin for RTL). Component is centered by default                                                                                                                                               |
+| className?         | string                         | Permits `css` prop to be applied                                                                                                                                                                                                                                             |
+| htmlAttributes?    | string                         | Permits custom html attributes to be applied. E.g. `data-ignore-lite` for .lite                                                                                                                                                                                              |
+| eventTrackingData? | shape({componentName: string}) | Tracking data needed for ati to track view and click of the Call to Action Link                                                                                                                                                                                              |
+| download?          | boolean                        | Permits download                                                                                                                                                                                                                                                             |
+| fontVariant?       | string                         | Used to render the correct `font-family`, `font-style` and `font-weight`. Possible values are `sansRegular`, `sansRegularItalic`, `sansBold`, `sansBoldItalic`, `sansLight`, `serifRegular`, `serifMedium`, `serifMediumItalic`, `serifBold`, `serifLight`.                  |
+| size?              | string                         | The [BBC GEL size](https://www.bbc.co.uk/gel/features/typography#type-sizes). Possible values are `atlas`, `elephant`, `imperial`, `royal`, `foolscap`, `canon`, `trafalgar`, `paragon`, `doublePica`, `greatPrimer`, `bodyCopy`, `pica`, `longPrimer`, `brevier`, `minion`. |
```

Comment:
As above, instead of listing them here, should we link to where they are defined as a type.